@xen-orchestra/lite/src/App/index.tsx

```diff
@@ -166,6 +166,15 @@ const mdTheme = createTheme({
       main: '#ffc107',
     },
   },
+  typography: {
+    fontFamily: 'inter',
```

Comment:
This can't work like that, right? We need to install and import the right font first.

Reply:
Yes and this was a temporary change while waiting for this [PR](https://github.com/vatesfr/xen-orchestra/commit/e5c737cba73b075e0a6f8866d3621d4549b4c1aa) to be merged.
I will remove this one and rebase on `xo-lite`